scripts/noshake.json

```diff
@@ -303,7 +303,7 @@
   "Mathlib.Init.Data.Nat.GCD": ["Batteries.Data.Nat.Gcd"],
   "Mathlib.Init.Data.List.Basic": ["Batteries.Data.List.Basic"],
   "Mathlib.Init.Data.Int.Basic": ["Batteries.Data.Int.Order"],
-  "Mathlib.Init.Data.Bool.Lemmas": ["Mathlib.Tactic.AdaptationNote"],
+  "Mathlib.Data.Bool.Basic": ["Mathlib.Tactic.AdaptationNote"],
```

Comment:
I think you can remove this line instead

Reply:
```suggestion
```